pkg/grpc/server.go

```diff
@@ -160,6 +166,10 @@ func (s *server) getMiddlewareOptions() []grpc_go.ServerOption {
 	if diag_utils.IsTracingEnabled(s.tracingSpec.SamplingRate) {
 		s.logger.Info("enabled gRPC tracing middleware")
 		intr = append(intr, diag.GRPCTraceUnaryServerInterceptor(s.config.AppID, s.tracingSpec))
+
+		if s.proxy != nil {
+			intrStream = append(intrStream, diag.GRPCTraceStreamServerInterceptor(s.config.AppID, s.tracingSpec))
```

Comment:
OK, so you are using the middleware code to intercept the gRPC calls and decide to proxy to the app. Nice.